hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/main/java/org/apache/hadoop/yarn/server/federation/store/records/SubClusterIdInfo.java

```diff
@@ -84,4 +84,9 @@ public boolean equals(Object obj) {
   public int hashCode() {
     return new HashCodeBuilder().append(this.id).toHashCode();
   }
+
+  @Override
+  public String toString() {
+    return "SubClusterIdInfo{ id='" + id + '\'' + '}';
```

Comment:
Let's do a single concat:
```
return "SubClusterIdInfo{ id='" + id + "'}";
```

Reply:
Thank you for your help to review the code! I will modify the code.